testcases/nameTest/subpackages_test.py

```diff
@@ -528,12 +396,7 @@ def setCSCH(self):
                 else:
                     self.csch = OpenJdk11DebugFc()
                     return
-            elif int(rpms.getMajorVersionSimplified()) == 12:
-                if self.getCurrentArch() in gc.getArm32Achs():
-                    self.csch = OpenJdk12()
-                    return
-                self.csch = OpenJdk12Debug()
-                return
+
```

Comment:
The newline appears few times in those commits, why so?

Reply:
I forgot about those; it should be ok now :)